source/_docs/migrate-wordpress.md

```diff
@@ -51,6 +46,21 @@ If the migration is not successful, contact <migrations@pantheon.io> and include
 The <a href="https://wordpress.org/plugins/wp-native-php-sessions/">WordPress Native PHP Sessions</a> plugin is automatically installed during the migration process. For more details on this plugin, see <a href="/docs/wordpress-sessions/">WordPress and PHP Sessions</a>.
 </div>
 
+## Troubleshooting
+### Cannot Login Using a Google Account
+Attempts to login with Google will fail if your organization uses Single Sign-On, resulting in the following error:
+
+![Migration Authentication Error](/source/docs/assets/images/migration-authentication-error.png)
+
+Simply press your browser's back button to re-authenticate by entering your email address and complete sign-on with your SAML Identity Provider. For details, see [Single Sign-On for Pantheon Organizations](/docs/sso-organizations/).
+
+### Destination Site Not Found Error
+If you are logged in with one identity and re-authenticate a different account then the site created will be associated with one account and the machine token with another, resulting in the following error:
+
+![Destination site not found](/source/docs/assets/images/bv-destination-not-found-error.png)
+
+Press your browser's back button from the Pantheon Dashboard and re-authenticate the user account for your current session.
```

Comment:
@ari-gold This worked for my test - can you confirm it's the solution we want to recommend?